### PR TITLE
feat(http-logger): support to concat multiple log with separator.

### PR DESCRIPTION
**Pull request by membphis**
### What this PR does / why we need it:

as title.

support to report log in this way:

```shell
curl http://****/log -d '
{xxxxxxxxxxxxx}
{yyyyyyyyyyyyy}
'
```

the request logs is concat by '\n' way.

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

**Comment by nic-chen** [comment]
I have a doubt, why not use a array json, but concat by '\n' ?

**Comment by nic-chen** [comment]
> I have a doubt, why not use a array json, but concat by '\n' ?

ignore that, I see it supports two ways..

File: apisix/plugins/http-logger.lua [hunk 1]
```diff
@@ -110,13 +113,6 @@ local function send_http_data(conf, log_message)
             .. "body[" .. httpc_res:read_body() .. "]"
     end
 
-    -- keep the connection alive
-    ok, err = httpc:set_keepalive(conf.keepalive)
```
Comment (nic-chen) [inline apisix/plugins/http-logger.lua:114]: is it a default action ? although `keepalive` is not in the plugin's schema.

Comment (membphis) [reply apisix/plugins/http-logger.lua:114]: the old code is wrong. that is why I delete them.

Comment (membphis) [reply apisix/plugins/http-logger.lua:114]: if we want to call `set_keepalive`, need to define them first.

**Review by gxthrj — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

**Pull request merged**